src/Routes/RoutesCommon.tsx

```diff
@@ -56,6 +57,16 @@ export const routes: NestedRoute[] = [
     mobileNav: false,
     disable: !getConfig().REACT_APP_FEATURE_THORCHAIN_LENDING,
   },
+  {
+    path: '/pools',
+    label: 'navBar.pools',
+    icon: <RiExchangeFundsLine />,
+    main: PoolsPage,
+    category: RouteCategory.Featured,
+    priority: 4,
```

Comment:
what are these priorities for? seeing several using `4`, just curious

Reply:
Seems to be used for mobile sorting of nav routes only, summoning @reallybeard on whether or not we want to change it

Reply:
I'll merge this as is, we can fix the `priority` if @reallybeard thinks it's needed 🙏